### PR TITLE
test: workaround python3-linux-procfs and networkmanager being broken

### PR DESCRIPTION
**Pull request by jelly**
In rawhide python3-linux-procfs can't be installed after the Python 3.11
rebuild. The package was not rebuild yet, the maintainer has been made
aware of the breakage. To keep our rawhide CI running allow the
installation of tuned to fail and skip the test on rawhide.

### CODE REVIEW

**Comment by jelly** [comment]
Great, rawhide is back up, bad it's totally borked:

```
GENERAL.DEVICE:                         cockpit1
GENERAL.TYPE:                           ethernet
GENERAL.HWADDR:                         FE:8C:6F:F0:77:81
GENERAL.MTU:                            1500
GENERAL.STATE:                          100 (connected)
GENERAL.CONNECTION:                     cockpit1
GENERAL.CON-PATH:                       /org/freedesktop/NetworkManager/ActiveConnection/3
WIRED-PROPERTIES.CARRIER:               on
IP4.GATEWAY:                            --
IP6.GATEWAY:                            --

Traceback (most recent call last):
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/verify/check-networkmanager-team", line 44, in testBasic
    self.wait_for_iface(iface1)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/verify/netlib.py", line 163, in wait_for_iface
    raise e
    ^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/verify/netlib.py", line 159, in wait_for_iface
    self.browser.wait_in_text(sel, text)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/common/testlib.py", line 621, in wait_in_text
    self.wait_js_cond("ph_in_text(%s,%s)" % (jsquote(selector), jsquote(text)),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/common/testlib.py", line 564, in wait_js_cond
    self.raise_cdp_exception("timeout\nwait_js_cond", cond, result["exceptionDetails"], trailer)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/common/testlib.py", line 270, in raise_cdp_exception
    raise Error("%s(%s): %s" % (func, arg, msg))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
testlib.Error: timeout
wait_js_cond(ph_in_text("#networking-interfaces tr[data-interface='cockpit1']","10.111.")): timeout

Wrote screenshot to TestTeam-testBasic-fedora-36-localhost-22-FAIL.png
Wrote HTML dump to TestTeam-testBasic-fedora-36-localhost-22-FAIL.html
Journal extracted to TestTeam-testBasic-fedora-36-localhost-22-FAIL.log.gz
> warning: transport closed: disconnected
Exiting due to channel error.
Exiting due to channel error.
Exiting due to channel error.
Exiting due to channel error.
Exiting due to channel error.
Traceback (most recent call last):
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/verify/check-networkmanager-team", line 44, in testBasic
    self.wait_for_iface(iface1)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/verify/netlib.py", line 163, in wait_for_iface
    raise e
    ^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/verify/netlib.py", line 159, in wait_for_iface
    self.browser.wait_in_text(sel, text)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/common/testlib.py", line 621, in wait_in_text
    self.wait_js_cond("ph_in_text(%s,%s)" % (jsquote(selector), jsquote(text)),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/common/testlib.py", line 564, in wait_js_cond
    self.raise_cdp_exception("timeout\nwait_js_cond", cond, result["exceptionDetails"], trailer)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/ARTIFACTS/work-allHASTux/plans/all/discover/default/tests/test/common/testlib.py", line 270, in raise_cdp_exception
    raise Error("%s(%s): %s" % (func, arg, msg))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
testlib.Error: timeout
wait_js_cond(ph_in_text("#networking-interfaces tr[data-interface='cockpit1']","10.111.")): timeout
```

**Comment by jelly** [comment]
Note, networkmanager requiresa bug report.

File: test/browser/browser.sh [hunk 1]
```diff
@@ -34,6 +34,9 @@ if grep -q 'ID=.*fedora' /etc/os-release; then
     dnf install -y tcsh
 fi
 
+# HACK: python3-linux-procfs can't be installed on rawhide
```
Comment (marusak) [inline test/browser/browser.sh:37]: Isn't this reported in BZ?

File: test/browser/run-test.sh [hunk 2]
```diff
@@ -87,23 +87,24 @@ if [ -n "$test_basic" ]; then
...
         TestLogin
-        TestNetworking
         TestServices
         TestSOS
         TestSystemInfo
-        TestTeam
         "
 
     # HACK: python3-linux-procfs can't be installed on rawhide
+    # HACK: NetworkManager 1.39.8 IpConfig.Addresses is an empty list
```
Comment (marusak) [inline test/browser/run-test.sh:100]: Seems dangerous just to skip it, if it is not reported as it will trickle down into rest of Fedora soon

**Comment by martinpitt** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=2098991 says it's allegedly fixed. Let's validate in https://github.com/cockpit-project/bots/issues/3552 and then drop the workaround here. Also, the NM regression should be a naughty IMHO.

**Comment by martinpitt** [comment]
https://github.com/cockpit-project/bots/pull/3552 succeeded, so tuned is installable again.

**Comment by martinpitt** [comment]
The last [rawhide run](https://artifacts.dev.testing-farm.io/c778fc14-015a-4305-b68e-bcb7336de25f/) in https://github.com/cockpit-project/cockpit/pull/17537 succeeded tuned, only NM failed. Let's handle the latter through a naughty.

**Pull request closed without merging**